src/ert/gui/tools/plot/plot_window.py

```diff
@@ -166,6 +178,7 @@ def __init__(self, config_file: str, parent: QWidget | None):
 
     @Slot(int)
     def currentTabChanged(self, index: Any) -> None:
```

Comment:
Should `index` be typed to `int` here or will that break stuff? If it breaks stuff, should we add an assert on the type of index before using it?

Reply:
I definitely should be int, as the slot is connected to the signal;
```
void QTabWidget::currentChanged(int index)
```
Nice catch!